### ISSUE TITLE
ctix-exclude-next is not working.

### ISSUE THREAD

**Issue by smallmain** [imjuni/ctix#122]
in file `./base/boost/cloneable/interface.ts`:

```ts
/** @ctix-exclude-next */
export interface Cloneable<T = unknown> {
    /**
     * 克隆一个新对象并返回
     */
    [clone](host?: ICloneHost): T;
}

export interface ICloneHost {
    /**
     * 传入原始对象，返回克隆后的对象
     */
    getCache<T extends object>(original: T): T | undefined;

    /**
     * 缓存传入的克隆对象
     */
    setCache<T extends object>(original: T, cloned: T): T;
}
```

output:
```ts
export * from "./base/boost/cloneable/interface.js";
```

expect:
```ts
export { ICloneHost } from "./base/boost/cloneable/interface.js";
```

demo:
[js-core_poly_.zip](https://github.com/imjuni/ctix/files/14738680/js-core_poly_.zip)

version:
v2.4.0

**Comment by imjuni** [comment]
@smallmain Thank you for your feedback!, I will check about that :)

**Comment by smallmain** [comment]
> @smallmain If you look at the attached example config file, you will see that `generationStyle` is set to `default-alias-named-star`. Since `ctix` is the first to apply the configuration values, please change this configuration or add the comment `/** @ctix-generation-style default-alias-named-destructive */` at the top of the `interface.ts` file. See the [IN_DEPTH_GEN_STYLE.md](https://github.com/imjuni/ctix/blob/master/doc/IN_DEPTH_GEN_STYLE.md) file for more information. Thank you.

i see, thank you.

**Comment by smallmain** [comment]
> @smallmain If you look at the attached example config file, you will see that `generationStyle` is set to `default-alias-named-star`. Since `ctix` is the first to apply the configuration values, please change this configuration or add the comment `/** @ctix-generation-style default-alias-named-destructive */` at the top of the `interface.ts` file. See the [IN_DEPTH_GEN_STYLE.md](https://github.com/imjuni/ctix/blob/master/doc/IN_DEPTH_GEN_STYLE.md) file for more information. Thank you.

Hi, it still doesn’t seem to be working properly.

```ts
/** @ctix-generation-style default-alias-named-destructive */

/**
 * 语义版本号正则表达式
 */
/** @ctix-exclude-next */
export const regexp =
    /^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/u;

/**
 * 语义化版本号正则表达式分组含义
 */
export enum SemverRegexpGroup {
    Source = 0,
    Major = 1,
    Minor = 2,
    Patch = 3,
    Prerelease = 4,
    Meta = 5,
}
```

In the same demo, I added comments to the header of the file but did not remove them.

output:

```ts
export { regexp, SemverRegexpGroup } from"./utils/semver/regexp.js";
```

**Comment by imjuni** [comment]
@smallmain Thank you for your participant! I was fixed this issue in `v2.4.3` :)
